core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -368,11 +390,30 @@ sealed abstract class IO[+E, +A] { self =>
   final def zipWith[E1 >: E, B, C](that: IO[E1, B])(f: (A, B) => C): IO[E1, C] =
     self.flatMap(a => that.map(b => f(a, b)))
 
+  /**
+   * Sequentially zips this effect with the specified effect, combining the
+   * results into a tuple.
+   */
+  final def zip[E1 >: E, B](that: IO[E1, B]): IO[E1, (A, B)] =
```

Comment:
can you please call this `seq` (as the dual for `par`) and also create a corresponding `seqWith`
( see https://github.com/scalaz/scalaz-zio/pull/138#discussion_r205790288 )